app.rb

```diff
@@ -127,7 +127,7 @@ def create_rental
 
     rental = Rental.new(date, @books[selected_id_book], @people[selected_id_person])
     @rentals.push(rental)
-    puts 'Rental created successfully'
+    puts 'Rental is created successfully'
     run
```

Comment:
- [ ] When I try to add a rental, I get the following error. kindly fix this issue and make sure the rental add correctly.
>![image](https://user-images.githubusercontent.com/53402600/216410525-ab9aea50-0ef6-4433-baa9-176f36f5cd30.png)